ironfish/src/genesis/makeGenesisBlock.ts

```diff
@@ -50,7 +52,7 @@ export async function makeGenesisBlock(
   // It should end up with 0 coins.
   const genesisKey = generateKey()
   // Create a genesis note granting the genesisKey allocationSum coins.
-  const genesisNote = new NativeNote(genesisKey.public_address, allocationSum, info.memo)
+  const genesisNote = new NativeNote(genesisKey.public_address, allocationSum, '')
```

Comment:
We're not using the memo here anymore?

Reply:
flags.memo is still in the single allocation when an `allocations` CSV isn't passed. It didn't make sense to put it on this Note since this note is only ever decryptable by the `genesisKey` above. The `genesisKey` is just used to temporarily hold the genesis supply before it's sent to the `allocations`.